test/basic.html

```diff
@@ -206,6 +206,77 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          ensureDocumentHasFocus();
```

Comment:
(recap from offline discussion) I'll change these tests to spy on `.focus()` like in the PR you linked to earlier.